pkg/config/config.go

```diff
@@ -25,21 +25,18 @@ const (
 	defaultCurrentContext = "relaysh"
 )
 
-var (
-	defaultAPIURL = &url.URL{Scheme: "https", Host: "api.relay.sh"}
-	defaultUIURL  = &url.URL{Scheme: "https", Host: "app.relay.sh"}
-	defaultWebURL = &url.URL{Scheme: "https", Host: "relay.sh"}
-)
-
 var defaultContexts = map[string]APIContext{
 	"relaysh": {
 		Name:      "relaysh",
-		APIDomain: defaultAPIURL,
-		UIDomain:  defaultUIURL,
-		WebDomain: defaultWebURL,
+		APIDomain: &url.URL{Scheme: "https", Host: "api.relay.sh"},
+		UIDomain:  &url.URL{Scheme: "https", Host: "app.relay.sh"},
+		WebDomain: &url.URL{Scheme: "https", Host: "relay.sh"},
 	},
 	"dev": {
-		Name: "dev",
+		Name:      "dev",
+		APIDomain: &url.URL{Scheme: "http", Host: "relay-api.local:8080"},
```

Comment:
Hm. I wonder if this should actually be set here -- for open-source users this won't ever be available. I think we can probably revisit when we implement the context stuff more broadly though.

Reply:
Yeah, I know. I'm planning a series of micro-adjustments to this until we have broad context support in place. You might not like my next set of changes either...

Planning on:

1. Contextualizing token caching.
2. Add default staging context name.
3. Hardcoding staging URLs.
4. Adding a command to switch context.

Partly because I'm already tired of switching the domains by hand and redoing the auth each time. Partly to ensure we have better ease-of-use internally without requiring excessive setup and/or documentation.

In the long run, I'd prefer something like the `kubectl` context model. Including a means of adding the context externally. I'd prefer we have a way of adding (all of) our specific contexts without hardcoding them. And enable this for users that can define their own default contexts.

Reply:
Being a little more cavalier with the `relay dev` commands is probably fine.